portal-test/src/com/liferay/portal/kernel/service/test/SynchronousInvocationHandler.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
```

Comment:
@dantewang @ChiZelin 
Please move this class to `com.liferay.portal.kernel.test`.

Tina